pyatlan/model/credential.py

```diff
@@ -94,6 +94,16 @@ def to_credential(self) -> Credential:
         )
 
 
+class CredentialResponseList(AtlanObject):
+    """
+    Model representing a response containing a list of CredentialResponse objects.
+
+    Attributes:
+        records (List[CredentialResponse]): The list of credential records returned.
```

Comment:
We shouldn’t document each attribute of any Pydantic model this way. Instead, just use the `description` field (as you already did), e.g: `description="list of credential records returned."`:

```suggestion

```